cmd/gen-go-sample/inittree.go

```diff
@@ -291,7 +296,7 @@ func (t *initTree) print(w *bufio.Writer, g *generator, ind int) error {
 	return nil
 }
 
-func snakeToCapCamel(s string) string {
+func snakeToPascal(s string) string {
```

Comment:
nit: consider appending `Case` to the names of this function and the next

Reply:
I thought of this too. It makes some otherwise-reasonable lines unreasonably long :(